sections/home/TopSection.jsx

```diff
@@ -78,11 +80,13 @@ const JosefinTitle = styled(SText)`
   ${({ theme }) => `
     font-family: ${theme.font.josefin};
     margin: 1em 0 0 0;
+		max-width: 30vw;
   `};
   ${media(
     1200,
     `
         font-size: 2.5rem;
+				max-width: 100vw;
```

Comment:
I'm not sure this is changing the width of the text sections. Try changing the width of the first column section @mobile breakpoint?

Reply:
![image](https://user-images.githubusercontent.com/65516245/128112460-ef043c0c-52ba-4a62-bfbe-48bf7b7abeda.png)
vs setting wider width on tablet 
![image](https://user-images.githubusercontent.com/65516245/128112508-b57379df-4655-40b7-a85c-39ac3cb986b3.png)